includes/ProductSections/Manager.php

```
@@ -0,0 +1,342 @@
+    }
+
+    /**
+     * Check products block visibility settings by admin and vendor.
+     *
+     * @since 3.3.7
+     *
+     * @param AbstractProductSection $section_object
+     * @param array $store_settings
+     *
+     * @return bool
+     */
+    protected function is_visible( $section_object, $store_settings ) {
+        $customizer_settings = dokan_get_option( 'product_sections', 'dokan_appearance' );
+        // Check if current products section enabled by admin.
+        // for customizer settings default value is hide sections
+        if ( $section_object->get_show_in_customizer() &&
+            (
+                ! isset( $customizer_settings[ $section_object->get_section_id() ] ) ||
+                'on' === $customizer_settings[ $section_object->get_section_id() ]
+            )
+        ) {
+            return false;
+        }
+
+        // Check if current products section enabled by vendor.
+        // for setting fields, default value is show
+        if ( $section_object->get_show_in_settings() &&
+            (
+                isset( $store_settings[ $section_object->get_section_id() ] ) &&
+                'yes' !== $store_settings[ $section_object->get_section_id() ]
+            )
+        ) {
+            return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * Check products block visibility settings by admin and vendor.
+     *
+     * @since 3.3.7
+     *
+     * @param AbstractProductSection $section_object
+     *
+     * @return bool
+     */
+    protected function is_enabled( $section_object ) {
+        $customizer_settings = dokan_get_option( 'product_sections', 'dokan_appearance' );
+        // for customizer settings default value is hide sections
+        if ( $section_object->get_show_in_customizer() &&
+            (
+                ! isset( $customizer_settings[ $section_object->get_section_id() ] ) ||
+                'on' === $customizer_settings[ $section_object->get_section_id() ]
```

Is this supposed to be `'on'` or `'off'`?

---

it is supposed to be on, // for customizer settings default value is hide sections